test/functional/3rd-party/3rd-party-bundle-add-symlinkc.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+
+# Author: Otavio Pontes
+# Email: otavio.pontes@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -n upstream-bundle -f /upstream_file "$TEST_NAME"
+
+	# create a couple 3rd-party repos within the test environment and add
+	# some bundles to them
+	create_third_party_repo -a "$TEST_NAME" 10 1 test-repo1
+
+	files_dir="$TEST_NAME"/3rd-party/test-repo1/10/files/
+	sudo ln -s broken "$files_dir"/symlink
+	hashed_name=$("$SWUPD" hashdump --quiet "$files_dir"/symlink)
+	sudo mv "$files_dir"/symlink "$files_dir"/"$hashed_name"
+
+	create_tar "$files_dir"/"$hashed_name"
+
+	create_bundle -n test-bundle1 -f /foo/symlink:"$files_dir"/"$hashed_name" -u test-repo1 "$TEST_NAME"
```

Comment:
Just a tip for the future, testlib supports the creation of bundles with dangling links with the use of the `-b` flag, it would have saved you some effort:
```
Usage:
    create_bundle [-L] [-t] [-e] [-n] <bundle_name> [-v] <version> [-u] <3rd-party repo> [-d] <list of dirs> [-f] <list of files> [-l] <list of links> ENV_NAME

Options:
    -L    When the flag is selected the bundle will be 'installed' in the target-dir, otherwise it will only be created in content dir
    -t    The bundle will be 'tracked' in the state-dir regardless of if its content is installed on the system
    -e    The bundle will be marked as experimental
    -n    The name of the bundle to be created, if not specified a name will be autogenerated
    -v    The version for the bundle, if non selected version 10 will be used
    -u    If the bundle is to be created inside a 3rd-party repo, the repo's name should be specified here
    -d    Comma-separated list of directories to be included in the bundle
    -f    Comma-separated list of files to be created and included in the bundle
    -l    Comma-separated list of symlinks to files to be created and included in the bundle
    -c    Comma-separated list of symlinks to directories to be created and included in the bundle
    -b    Comma-separated list of dangling (broken) symlinks to be created and included in the bundle
```

Reply:
Nice, I didn't find that when I was looking for it. I'll use that in this PR